packages/now-python/now_init.py

```diff
@@ -3,9 +3,13 @@
 import base64
 import json
 import inspect
+from importlib import util
 
-import __NOW_HANDLER_FILENAME
-__now_variables = dir(__NOW_HANDLER_FILENAME)
+# Import relative path https://stackoverflow.com/a/67692/266535
+__now_spec = util.spec_from_file_location("__NOW_HANDLER_MODULE_NAME", "./__NOW_HANDLER_ENTRYPOINT")
```

Comment:
Is this `./` going to work on Windows?

Reply:
Good question. I wish our `now dev` tests were working with Windows 😖 #3727 

I'll have to try it out

Reply:
I confirmed this import code works on Windows 👍